.github/workflows/build.yml

```diff
@@ -148,24 +160,39 @@ jobs:
           
       - name: Run unit tests
         run: npm run test:unit
+        env:
+          CI: true
         working-directory: ./src/vscode-bicep
-        
+
       - name: Run E2E tests
+        run: npm run test:e2e
+        env:
+          CI: true
+        if: runner.os != 'Linux'
+        working-directory: ./src/vscode-bicep
+        
+      - name: Run E2E tests (Linux)
         run: xvfb-run -a npm run test:e2e
+        env:
+          CI: true
+        if: runner.os == 'Linux'
         working-directory: ./src/vscode-bicep
 
       - name: Upload Code Coverage
         uses: codecov/codecov-action@v1
+        if: runner.os == 'Linux'
```

Comment:
>== [](start = 22, length = 2)

Why limit code coverage to just Linux machines? We can upload the report from all the jobs and codecov will merge.

Reply:
Good to know that codecov will merge them. I'll remove the if condition.